css/properties/zoom.json

```diff
@@ -59,10 +59,14 @@
             "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/zoom#Values",
             "support": {
               "chrome": {
-                "version_added": null
+                "version_added": true,
+                "version_removed": "59",
+                "notes": "See <a href='https://chromium.googlesource.com/chromium/src/+/30a210d86eaf8ca5bf2941a6bbee44b867300c6b'>this commit</a>."
```

Comment:
Sometimes we link to bugs or release notes related to deprecated or removed features, but rarely corresponding commits (it's a lot of extra detail). For a removed feature, it's of especially little use to readers. Let's drop these notes.

Reply:
All the commit notes are droped, thanks for hinting.